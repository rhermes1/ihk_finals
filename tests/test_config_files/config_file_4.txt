; *********************************
; Gebiet ohne Stroemung. 
; *********************************
; Groesse des Gebietes
35 30
; zu fahrende Route
5 30 25 5 
; Stroemungen
0 0 1 1 
