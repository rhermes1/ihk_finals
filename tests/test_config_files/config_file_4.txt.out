*********************
Gebiet ohne Stroemung
*********************
Groesse des Gebietes
 35.0 30.0

Stroemungen
 (0.0;0.0) (35.0;30.0) SV = (0.0;0.0)

zu fahrende Route
 Punkt 0: (5.0;30.0)

 Punkt 1: (25.0;5.0)


Daten fuer die Planung der Route:
 Strecke von (5.0;30.0) nach (25.0;5.0)
  Entfernung: 32.0 km
  SKV = (3.1;-3.9)
  Sollgeschwindigkeit: 5.0 km/h
  Fahrzeit: 6.4 h
